hr-data-generator/src/main/java/uk/gov/gchq/palisade/example/hrdatagenerator/CreateDataFile.java

```diff
@@ -42,42 +45,41 @@ public CreateDataFile(final long numberOfEmployees, final long seed, final File
     }
 
     public Boolean call() {
-        outputFile.getParentFile().mkdirs();
-        try (OutputStream out = new FileOutputStream(outputFile)) {
-            Stream<Employee> employeeStream;
-            AvroSerialiser<Employee> employeeAvroSerialiser = new AvroSerialiser<>(Employee.class);
-            // Need one Employee whose manager has a UID of Bob (for examples to work)
-            Employee firstEmployee = Employee.generate(random);
-            Manager[] managers = firstEmployee.getManager();
-            UserId lineManagerUid = managers[0].getUid();
-            lineManagerUid.setId("Eve");
-            managers[0].setUid(lineManagerUid);
-            firstEmployee.setManager(managers);
-            Stream<Employee> firstEmployeeStream = Stream.of(firstEmployee);
-            if (numberOfEmployees > 1) {
-                Stream<Employee> moreEmployeesStream = generateStreamOfEmployees();
-                employeeStream = Stream.concat(firstEmployeeStream, moreEmployeesStream);
-            } else {
-                employeeStream = firstEmployeeStream;
-            }
+        boolean mkdirSuccess = outputFile.getParentFile().mkdirs();
+        if (mkdirSuccess) {
```

Comment:
no point setting a variable to only use it once. replace with: if (outputFile.getParentFile().mkdirs())

Reply:
I was kind-of halfway on it, since the line has a side-effect of creating  directories i thought it best to keep it separate and assign a variable